src/utils/xmltools.ts

```diff
@@ -124,7 +124,8 @@ function formatField(field: GeneratedField): string {
   const optional = (field.optional && !isTypeBoolean) ? "?" : "";
   const comment = field.comment ? formatComment(field.comment) : "";
   const type = formatType(field.type, field.subfields || []);
-  return `${comment}  ${field.name}${optional}: ${type};`;
+  const hasDash = field.name.indexOf('-') > -1;
+  return `${comment}  ${hasDash ? "'" : ""}${field.name}${hasDash ? "'" : ""}${optional}: ${type};`;
```

Comment:
Could put the single quote/empty string in a variable to unclutter the template.